span-processing-utils/src/main/java/org/hypertrace/config/span/processing/utils/SpanFilterMatcher.java

```diff
@@ -111,7 +111,12 @@ private boolean matches(String lhs, String rhs, RelationalOperator relationalOpe
       case RELATIONAL_OPERATOR_ENDS_WITH:
         return lhs.endsWith(rhs);
       case RELATIONAL_OPERATOR_REGEX_MATCH:
-        return Pattern.compile(rhs).matcher(lhs).find();
+        try {
+          return Pattern.compile(rhs).matcher(lhs).find();
+        } catch (Exception e) {
+          log.error("Invalid regex passed to match: {}", e);
```

Comment:
Can we also log the regex? And, can we also add a debug log (if debug is enabled) to log both lhs and rhs?

Reply:
Okay